docs/references/errors.md

```diff
@@ -7,7 +7,7 @@ slug: errors
 
 This page contains the list of error codes emitted by Restate components.
 
-## META0001
+## META0001 {#META0001}
```

Comment:
Didn't it work without these tags? It should by using `/references/errors#meta0001`, I think?

Reply:
I guess it is generated by https://github.com/restatedev/documentation/blob/main/tools/generate_errors_page.sh#L31. Seems to be there to generate anchors that are capital letters.